### DISCUSSION TITLE
Where exactly "register models with prefixed ids" by putting "Spatie\PrefixedIds\PrefixedIds::registerModels([ ... ])" code ???

### DISCUSSION THREAD

**Discussion by chegmarco1989** [spatie/laravel-prefixed-ids#17] [Q&A]
Hello.

After put this:

```
// Registering models with prefixed ids:
Spatie\PrefixedIds\PrefixedIds::registerModels([
	'PUBLIC-Key_' => Publicapikey::class,
	'PRIVATE-Key_' => Privateapikey::class,
	'SECRET-Key_' => Secretapikey::class,
]);
```

Into: `App\Providers\AppServiceProvider.php` exactly like this:

![image](https://user-images.githubusercontent.com/51763985/221709033-36b9a9b1-d839-4b87-91f1-55492d041349.png)

We got this error:

![image](https://user-images.githubusercontent.com/51763985/221709450-3dd59bdf-cdb0-4dcb-8d6c-ffd8097794dc.png)

So, how to fix this error or Where exactly **register models with prefixed ids** by putting `Spatie\PrefixedIds\PrefixedIds::registerModels([ ... ])` code ???

Please Help.

**Comment by danjohnson95** [comment]
You're missing the `use` statement at the top of the file.

Add this in:

```php
use Spatie\PrefixedIds\PrefixedIds;
```

and replace the usage with `PrefixedIds::registerModels([`

or you could just stick a `\` before `Spatie` on line 27.

```php
\Spatie\PrefixedIds\PrefixedIds::registerModels([
```
